### PR TITLE
Add WRF case study

### PR DESCRIPTION
**Pull request by yzanhua**
A WRF.md file is created to explain how to use WRF with Log Vol. An example timing results on Cori is also provided.

### CODE REVIEW

**Comment by wkliao** [comment]
There is no need to use a jpg file for the table at the bottom.
Check how to add a table in an md file.

File: case_studies/README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ## Case Studies
 * [E3SM I/O kernel](./E3SM_IO.md) - Performance study of the I/O kernel of [E3SM](https://github.com/E3SM-Project/E3SM) and comparison with other I/O methods.
+* [WRF](./WRF.md) - Performance study of the [WRF](https://github.com/wrf-model/WRF) applicaiton and comparison with other I/O methods.
```
Comment (wkliao) [inline case_studies/README.md:3]: Remove "and comparison with other I/O methods." as we do not have that.

**Comment by wkliao** [comment]
Please rebase this branch and resolve the conflicts.

**Comment by yzanhua** [comment]
This branch is rebased on to master branch

**Pull request merged**